scipy/stats/_multivariate.py

```diff
@@ -5739,3 +5749,295 @@ def _sample_uniform_direction(dim, size, random_state):
     samples = random_state.standard_normal(samples_shape)
     samples /= np.linalg.norm(samples, axis=-1, keepdims=True)
     return samples
+
+
+class dirichlet_multinomial_gen(multi_rv_generic):
+    r"""A Dirichlet multinomial random variable.
+
+    Methods
+    -------
+    logpmf(alpha, x):
```

Comment:
For consistency with all other distributions in SciPy, shape parameters must be consistent for all methods within a distribution: `logpmf` and `pmf` need the `n` parameter, and `n` should be accepted when freezing the distribution. If `n` and `x` do not agree, the probability is zero.

Reply:
For consistency with all other distributions in SciPy, the `x` argument must go first, then the shape paramters.